client/file-manager/src/rocksdb.rs

```diff
@@ -497,6 +502,19 @@ where
     DB: KeyValueDB,
     HasherOutT<T>: TryFrom<[u8; H_LENGTH]>,
 {
+    /// Helper to build the bucket-prefixed file key used for efficient prefix scans.
```

Comment:
I believe it would be more appropriate to mention that we add the bucket prefix to be able to efficiently delete all files given some bucket id